config.go

```diff
@@ -234,7 +236,10 @@ type config struct {
 	NoSeedBackup bool `long:"noseedbackup" description:"If true, NO SEED WILL BE EXPOSED AND THE WALLET WILL BE ENCRYPTED USING THE DEFAULT PASSPHRASE -- EVER. THIS FLAG IS ONLY FOR TESTING AND IS BEING DEPRECATED."`
 
 	TrickleDelay        int           `long:"trickledelay" description:"Time in milliseconds between each release of announcements to the network"`
-	InactiveChanTimeout time.Duration `long:"inactivechantimeout" description:"If a channel has been inactive for the set time, send a ChannelUpdate disabling it."`
+	ChanActiveTimeout   time.Duration `long:"chan-active-timeout" description:"The duration that a peer connection must be stable before attempting to send a channel announcement that enables or reenables a given channel on the network."`
+	ChanInactiveTimeout time.Duration `long:"chan-inactive-timeout" description:"The duration that must elapse after first detecting that an already active channel is actually inactive and the channel announcement disabling the channel is announced to the network."`
+
+	ChanStatusSampleInterval time.Duration `long:"chan-status-sample-interval" description:"The polling interval between attempts to detect if an active channel has become inactive with respect to the channel's ability to forward."`
```

Comment:
Do we really want to make this configurable? Do you think it will provide value to the user?

Reply:
I don't think it hurts to expose it since we validate it properly. If the user wants to set super short/long timeouts then the default interval may be inappropriate.

Reply:
It just bloats the exposed config, and seems like all three could be derived from just setting the longest one, even during integration tests.

Reply:
still in favor of giving users full control , this is an important piece of allowing routing node operators to start specifying uptime SLAs for their peers. saying you want 90% uptime over a day has very diffent meanings depending on the sample interval. any deterministic function will produce unsatisfactory values for a subset of users and timescales.